ansible/roles/donkeycar/tasks/main.yml

```
@@ -64,7 +64,13 @@
   become_user: pi
   lineinfile:
     dest: /home/pi/.bashrc
-    line: "source ~/venv/bin/activate"
+    line: 'source ~/venv/bin/activate'
+
+- name: Replace numpy version in setup.py
+  ansible.builtin.lineinfile:
+    path: /home/pi/projects/setup.py
+    search_string: 'numpy==1.19'
+    replace: 'numpy==1.21.6'
```

```suggestion
    line: 'numpy==1.21.6'
```
`replace` doesn't exists, see [error in packer execution](https://github.com/mdl29/donkeycarLPH/actions/runs/5164361800/jobs/9303209051#step:7:316), you should use `line` [according to the documentation](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/lineinfile_module.html#parameter-line)